src/widgets/LastSensorDownload.js

```diff
@@ -72,6 +73,8 @@ const localStyles = StyleSheet.create({
   headerTextWithIcon: {
     flex: 0,
     paddingHorizontal: 8,
+    width: 150,
```

Comment:
Again not sure about this set width. Think it looks OK when the `last download time` is not `N/A`, but when it is it looks silly - but that won't be for long, usually, so..?

Reply:
is this to make the battery icons align vertically?
if so - are we allowed to change the order, and have this on the left of the battery? The text for the battery is `100%` - `1%` so there's very little width change

Reply:
75307e0

Reply:
great idea